.github/workflows/pubsys-exercise.yaml

```
@@ -0,0 +1,27 @@
+name: pubsys
+on:
+  pull_request:
+    branches: [develop]
+    paths:
+      - 'tools/**'
```

```suggestion
      - 'tools/pubsys*/**'
    paths-ignore:
      - '**.md'
      - '**.example'
      - '**.template'
      - 'pubsys/policies/**'
```
We shouldn't need to test buildsys (or future non-pubsys systems) in the pubsys workflow, and should probably ignore things that don't affect the test.

---

Apparently you're not allowed to use `paths` and `paths-ignore` in the same workflow event: https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#excluding-paths. It has to be one or the other.

---

Can use `!` in paths to exclude paths.

---

@webern also pointed out that we should run on changes to Makefile.toml, since that can change the targets we're calling in the workflow.